<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelegramWindow</class>
 <widget class="QDialog" name="TelegramWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Username">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>17</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Usercode">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>47</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Usercode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Chat">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>77</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chat:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_username_set">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>17</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_usercode_set">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>45</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>usercode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Chat_set">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>77</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>chat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>107</y>
     <width>140</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
